src/Avalonia.X11/X11Platform.cs

```diff
@@ -162,14 +162,48 @@ bool ShouldUseXim()
 
 namespace Avalonia
 {
-
+    /// <summary>
+    /// Platform-specific options which apply to Linux.
+    /// </summary>
     public class X11PlatformOptions
     {
+        /// <summary>
+        /// Enables native Linux EGL when set to true. The default value is false.
+        /// </summary>
         public bool UseEGL { get; set; }
+
+        /// <summary>
+        /// Determines whether to use GPU for rendering in your project. The default value is true.
+        /// </summary>
         public bool UseGpu { get; set; } = true;
+
+        /// <summary>
+        /// Embeds popups to the window when set to true. The default value is false.
+        /// </summary>
         public bool OverlayPopups { get; set; }
+
+        /// <summary>
+        /// Enables global KDE menu. The default value is false.
```

Comment:
Technically that's dbusmenu / appmenu protocol that originated in Ubuntu Unity desktop environment and was then adopted by KDE (included out of the box) and is available for XFCE, MATE and ValaPanel (for LDXE).

Reply:
tbf the only prominent thing that uses dbusmenu nowadays is KDE so this is alright IMO

Reply:
but we can generalize to dbusmenu and say which major DE's uses it still